core/jsMain/src/mathJs.kt

```diff
@@ -58,3 +58,35 @@ internal actual fun safeMultiply(a: Int, b: Int): Int {
     if (result > Int.MAX_VALUE || result < Int.MIN_VALUE) throw ArithmeticException("Multiplication overflows Int range: $a * $b.")
     return result.toInt()
 }
+
+/**
+ * Returns the floor division.
+ * <p>
+ * This returns {@code 0} for {@code floorDiv(0, 4)}.<br />
+ * This returns {@code -1} for {@code floorDiv(-1, 4)}.<br />
+ * This returns {@code -1} for {@code floorDiv(-2, 4)}.<br />
+ * This returns {@code -1} for {@code floorDiv(-3, 4)}.<br />
+ * This returns {@code -1} for {@code floorDiv(-4, 4)}.<br />
+ * This returns {@code -2} for {@code floorDiv(-5, 4)}.<br />
```

Comment:
Do not use `{@code}` and `<br>` tags in kdocs.

Reply:
Fixed, and also removed such constructs where they were copied from.